README.md

```diff
@@ -74,39 +44,173 @@ This program was created with the following tech stack and packages:
+    3. Move the folder into the `speaking-portal-project` directory
+    4. Delete the `extras` folder which is currently located at `speaking-portal-project\rhubarb\extras`
+5. In the terminal, navigate to `speaking-portal-project`
+6. Run `npm install` to ensure all packages are installed
+7. Run `npm start` to start the API
+8. Check in the terminal that the API is listening
+9. Create the user's Kukarella files:
     1. Go to [Kukarella](https://www.kukarella.com/)
-    2. Enter text, select a language, select a voice, and convert to audio
+    2. Enter text, select a language, select a voice, and convert the text to audio
     3. Download the audio as a .wav file
     4. Download the text file
-4. Move the Kukarella files to `speaking-portal-project\rhubarb`
-5. In the terminal, navigate to `speaking-portal-project`
-6. Install ffmpeg
-   1. Follow the steps on [adamtheautomator.com](https://adamtheautomator.com/install-ffmpeg/) for your corresponding
-   operating system. We recommend the Powershell approach for Windows users.
-7. Run `npm install`
-8. Run `npm start`
+10. Use Postman or a similar platform to send a POST request (see [POST Request Properties](#post-request-properties))
+to the local API that is currently listening for a request from step 7.
+11. Wait for the API to process the request and return the video
+12. Enjoy the animation!
 
 ## How it Works
 
-The Kurkarella TTS app creates the synthesized speech file from the user’s voice and text input. The Animation Add-On
-is broken down into two main components: the phoneme factory and the animation processor. The phoneme factory is in
-charge of mapping spoken language from an audio file into a series of phonemes, which are units of sound that
-distinguish one word from another. In order to make this process as accurate as possible, the phoneme factory also
-receives the user text file and language selection.
+### Overview
+
+The Speaking Portal Project (SPP) is built to connect with Kukurella's Text-to-Speech platform as an API. The SPP is broken down into three main components: *The *API**, **The Phoneme Factory**, and **The Animation Factory**.
+
+### API
+
+Speaking Portal API creates an MP4 output animation from a synthesized speech file and text input based on Kukarella
+TTS. Text and audio inputs are first processed and converted into a json file containing phonemes and their timings. This processing is done via the [Rhubarb library](https://github.com/DanielSWolf/rhubarb-lip-sync). (Phonemes are the
+smallest unit of sounds that uniquely identifies words from one another.) Once a phoneme file is created, it is
+processed in a phoneme processor, which maps phonemes to  mouth assets. The mouth assets are then added onto an avatar
+and rendered in a video file through ffmepg. The API, once request is received, initializes a node instance, creates a
+`/tmp` directory for file I/O operations, and begins the animation process by sending all user inputs to main.
```

Comment:
Technically this doesnt happen every time a request is recieved, when a request is received the files are stored within the already existing tmp directory and then the user inputs are read and processed

Reply:
resolving this conversation because Logan found a better way and we already committed it